tests/ops/models/test_privacy_request.py

```diff
@@ -74,6 +74,50 @@ def test_create_privacy_request_sets_requested_at(
     pr.delete(db)
 
 
+def test_create_privacy_request_sets_due_date(
+    db: Session,
+    policy: Policy,
+) -> None:
+    pr = PrivacyRequest.create(
+        db=db,
+        data={
+            "policy_id": policy.id,
+            "status": "pending",
+        },
+    )
+    assert pr.due_date is not None
+    pr.delete(db)
+
+    requested_at = datetime.now(timezone.utc)
+    due_date = timedelta(days=policy.execution_timeframe) + requested_at
+    pr = PrivacyRequest.create(
+        db=db,
+        data={
+            "requested_at": requested_at,
+            "policy_id": policy.id,
+            "status": "pending",
+        },
+    )
+    assert pr.due_date == due_date
+    pr.delete(db)
+
+    requested_at_str = "2021-08-30T16:09:37.359Z"
+    requested_at = datetime.strptime(requested_at_str, "%Y-%m-%dT%H:%M:%S.%fZ").replace(
```

Comment:
Then we could refer to that same constant here, and if for any reason we do decide to change the format, there's only one place to update